src/features/classification/Classification.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+import * as React from 'react';
+import Color from 'color';
+
+import Tooltip from '../../components/tooltip';
+import { bdlYellorange } from '../../styles/variables';
+
+import './Classification.scss';
+
+type Props = {
+    advisoryMessage?: string,
+    color?: string,
+    isMessageInline?: boolean,
+    name: string,
+};
+
+const Classification = ({ advisoryMessage, color = bdlYellorange, isMessageInline = true, name }: Props) => {
+    const borderColor = color;
+    const backgroundColor = Color(color).lighten(0.75);
```

Comment:
Should `Color` be lowercase, since it's not being used as a constructor?

Reply:
Using as per their documentation

Reply:
Its actually a class, `new` keyword is optional for instantiating anyways.